docs/install.md

```diff
@@ -380,6 +380,15 @@ agent = Agent(model)
 
 ### Ollama
 
-To use [Ollama](https://ollama.com/), you must first download the Ollama client, and then download a model.
+To use [Ollama](https://ollama.com/), you must first download the Ollama client, and then download a model using [Model Library](https://github.com/ollama/ollama?tab=readme-ov-file#model-library)
+
+```python {title="ollama_model_init.py"}
```

Comment:
well, this should come after the docs having the server running.

When we do already have an example on the [ollama model page](https://ai.pydantic.dev/api/models/ollama/). we should probably just link to that, rather than having examples in two places.

Reply:
But in the installation page, we have explanation of each methods so why not ollama. So if you want I can add the ollama run lines too.